### ISSUE TITLE
sql error on allocation_fetch

### ISSUE THREAD

**Issue by cnlangzi** [0chain/0chain#1451]
```

./zbox upload --localpath ~/Downloads/ImageOptim.tbz2 --allocation $alloc --remotepath /ImageOptim1.tbz2                                                                                                     11:09:38
0chain-core-sdk [INFO]   2022/06/29 11:16:48.876529 wallet.go:292: ******* Wallet SDK Version:v1.7.2-2-g3237f5c-dirty ******* (SetLogFile)
0box-sdk        [INFO]   2022/06/29 11:16:48.879494 sdk.go:73: ******* Storage SDK Version: v1.7.2-2-g3237f5c-dirty *******
0chain-core-sdk [INFO]   2022/06/29 11:16:51.159487 wallet.go:415: ******* Wallet SDK Version:v1.7.2-2-g3237f5c-dirty ******* (InitZCNSDK)
0box-sdk        [DEBUG]  2022/06/29 11:16:52.767852 networkworker.go:105: Get network result:{"miners":["https://test.0chain.net/miner03","https://test.0chain.net/miner02","https://test.0chain.net/miner01"],"sharders":["https://test.0chain.net/sharder01","https://test.0chain.net/sharder02"]}
Error fetching the allocation. allocation_fetch_error: Error fetching the allocation.internal_error: can't get allocation: error retrieving allocation: 8a274a48f4ff78adb24b8fa9a074832926ea33d20f941f37772655d46ea9fbda, error: sql: Scan error on column index 16, name "read_price_max": converting driver.Value type int64 ("-9223372036854775808") to a uint64: invalid syntax
```
detail https://0chain.slack.com/archives/C01GMABG2T1/p1656472583372719?thread_ts=1656464433.865429&cid=C01GMABG2T1

**Comment by lpoli** [comment]
@cnlangzi This is not blobber issue.
It is smartcontract issue in 0chain.

**Comment by cnlangzi** [comment]
let's transfer it to 0chain.

**Comment by cnlangzi** [comment]
it is duplicated with https://github.com/0chain/0chain/issues/1452

**Comment by cnlangzi** [comment]
fixed by #1448
